presto-docs/src/main/sphinx/release/release-0.147.rst

```diff
@@ -5,15 +5,67 @@ Release 0.147
 General Changes
 ---------------
 
+* Fix race condition that can cause queries that process data from non-columnar
+  data sources to fail.
+* Fix incorrect formatting of very early dates and timestamps.
+* Fix handling of syntax errors when parsing ``EXTRACT``.
+* Fix potential scheduling deadlock for connectors that expose node-partitioned data.
 * Add :doc:`/sql/show-create-view`.
+* Add :doc:`/sql/show-create-table`.
+* Add support for column aliases in ``WITH`` clause.
 * Support ``LIKE`` clause for :doc:`/sql/show-catalogs` and :doc:`/sql/show-schemas`.
+* Add support for ``INTERSECT``.
+* Add support for casting row types.
+* Add :func:`sequence` function.
+* Add :func:`sign` function.
+* Add :func:`flatten` function.
+* Add experimental implementation of :doc:`resource groups </admin/resource-groups>`.
+* Add ``query_priority`` session property, which can be used in combination with
+  resource groups.
+* Add :doc:`/connector/localfile`.
+* Remove experimental intermediate aggregation optimizer. The ``optimizer.use-intermediate-aggregations``
+  config option and ``task_intermediate_aggregation`` session property are no longer supported.
+* Add support for colocated joins for connectors that expose node-partitioned data.
+* Fix performance regression during planning.
```

Comment:
Move this to the top. Maybe phrase it like "Fix performance regression that increased planning time."